packages/hmr/index.mts

```diff
@@ -0,0 +1 @@
+export * from './lib/plugins/index.js';
```

Comment:
```suggestion
export * from './lib/plugins';
```

Reply:
unfortunately it turns out that such flavour of imports is required for ES Modules. I have the same compiling problem in my own repository

Reply:
it need to be like that because of `module`, about what i was writing above

Reply:
I'll see if I can't find a workaround, and is it possible to at least do a .ts? I think it's more confusing with .js.

Reply:
No, because of:
![image](https://github.com/user-attachments/assets/a884fa1d-d987-4d30-a0d2-79d4e1e20b2a)

But it can't be set, because of:
![image](https://github.com/user-attachments/assets/c08c2573-454a-43b5-94e2-24fb9fc6bf15)
![image](https://github.com/user-attachments/assets/c23fb391-5b66-4468-9c76-6c4a2f07584b)

And we can't change it to `bundler` XD

I was trying so many times to solve it, as i've described yesterday 😆

Reply:
I think we need to find a solution for .js, this is so confusing...

Reply:
Let's try, i can't do it more pretty